### PR TITLE
Fix typo in r2pm

### PR DESCRIPTION
**Pull request by bazuchan**

### CODE REVIEW

File: binr/r2pm/r2pm [hunk 1]
```diff
@@ -334,7 +334,7 @@ R2PM_GIT() {
 		cd "${DIR}" || R2PM_FAIL "Oops"
 	fi
 	r2 -qv | grep -q git || { # r2 >= 4.5
-		git checkout "r2-${R2_VERSION}"
+		git checkout "r2-${R2VERSION}"
```
Comment (trufae) [inline binr/r2pm/r2pm:337]: oh! i see, this variable is set at the begining of the script. but i think the right var name should be R2_VERSION. i would suggest using something like this:

```
$ eval $(r2 -H)
$ echo $R2_VERSION
4.6.0-git
$
```

**Review by trufae — CHANGES_REQUESTED** [review]
Can you try this eval line and update the PR? Nice catch

**Comment by trufae** [comment]
ping

**Comment by bazuchan** [comment]
will do tomorrow

File: binr/r2pm/r2pm (hunk 1 above)
Comment (zamaudio) [reply binr/r2pm/r2pm:337]: So is this PR a no-op then?

**Review by trufae — APPROVED** [review]

**Pull request merged**